sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/LoggingUtil.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.util;
+
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.core.util.logging.ClientLogger;
+
+/**
+ * Utilities to handle logging for credentials.
+ */
+public final class LoggingUtil {
+    /**
+     * Log a success message for a getToken() call.
+     * @param credential the credential class
+     * @param logger the logger to output the log message
+     * @param context the context of the getToken() request
+     */
+    public static void getTokenSuccess(Class<?> credential, ClientLogger logger, TokenRequestContext context) {
+        logger.info("Azure Identity => getToken() result for {}: SUCCESS", credential.getSimpleName());
+        logger.verbose("Azure Identity => Scopes: [{}]", String.join(", ", context.getScopes()));
+    }
+
+    /**
+     * Log an error message for a getToken() call.
+     * @param credential the credential class
+     * @param logger the logger to output the log message
+     * @param error the error thrown during getToken()
+     */
+    public static void getTokenError(Class<?> credential, ClientLogger logger, Throwable error) {
```

Comment:
It might be more useful to have the `context` on the error log than on success log. Logging the requested scope when an error occurs will help troubleshooting issues (unless the `Throwable` is guaranteed to have this scope information).

Reply:
Since it doesn't require users to call these methods, I'm adding `TokenRequestContext context` parameter to this method as well as the success log. I'll print the scopes in error log stream for `logTokenError` and verbose log stream for `logTokenSuccess`.